pages/k8s/operations.md

```diff
@@ -122,6 +122,27 @@ juju config kubernetes-worker ingress=true
 Ingress resources are DNS mappings to your containers, routed through
 [endpoints](https://kubernetes.io/docs/user-guide/services/).
 
+## Configuring CoreDNS
+
+CDK 1.14+ has CoreDNS enabled by default, which allows pods within Kubernetes
+to communicate with other pods or services by name.
+
+If you would like to disable CoreDNS, you can use the dns-provider option to do
```

Comment:
"...like to disable DNS (for example, to deploy your own custom DNS solution), you can use..."